sdk/javascript/README.md

```diff
@@ -0,0 +1,92 @@
+# Symbol-SDK
+
+[![lint][sdk-javascript-lint]][sdk-javascript-job] [![test][sdk-javascript-test]][sdk-javascript-job] [![vectors][sdk-javascript-vectors]][sdk-javascript-job] [![][sdk-javascript-cov]][sdk-javascript-cov-link] [![][sdk-javascript-package]][sdk-javascript-package-link]

```

Comment:
probably ok, but slightly different than what is on front page (no <br>)

Reply:
the only difference should be no table, for single project it looks bit better imo:

<img width="927" alt="image" src="https://user-images.githubusercontent.com/6421169/157464158-7f8efc30-1bd1-41dd-9c02-18d6f1d07fa0.png">